### PR TITLE
Fixes:#5101-Add support to enable HTTPS only for webapps

### PR DESCRIPTION
**Pull request by panchagnula**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5792
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -89,6 +89,7 @@ def load_arguments(self, _):
 
     with self.argument_context('webapp update') as c:
         c.argument('client_affinity_enabled', help="Enables sending session affinity cookies.", arg_type=get_three_state_flag(return_label=True))
+        c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.", arg_type=get_three_state_flag(return_label=True))
```
Comment (tjprescott) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py]: Is there a reason this is exposed on `webapp update` but not `webapp create`?  Does this apply to function apps?

Comment (panchagnula) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py]: our webapp create doesn't support setting any properties

Comment (panchagnula) [reply src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py]: there is no az functionapp update command, created the following to track & fix https://github.com/Azure/azure-cli/issues/5793

**Pull request merged**